diff --git a/node_modules/@sentry/react-native/RNSentry.podspec b/node_modules/@sentry/react-native/RNSentry.podspec
index 7c670d7..e1adac1 100644
--- a/node_modules/@sentry/react-native/RNSentry.podspec
+++ b/node_modules/@sentry/react-native/RNSentry.podspec
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
 
   s.preserve_paths = '*.js'
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
   s.dependency 'Sentry', '~> 5.2.0'
 
   s.source_files = 'ios/RNSentry.{h,m}'
