diff --git a/node_modules/react-native-mail/react-native-mail.podspec b/node_modules/react-native-mail/react-native-mail.podspec
index e31b682..41d4683 100644
--- a/node_modules/react-native-mail/react-native-mail.podspec
+++ b/node_modules/react-native-mail/react-native-mail.podspec
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.source          = { :git => "https://github.com/chirag04/react-native-mail", :tag => "v#{s.version}" }
   s.source_files    = 'RNMail/*.{h,m}'
   
-  s.dependency 'React'
+  s.dependency 'React-Core'
 
 end
